app/build.gradle

```diff
@@ -124,4 +125,8 @@ dependencies {
     implementation 'com.squareup.okhttp3:okhttp:3.12.13'
     implementation 'com.squareup.okio:okio:1.17.5'
     implementation 'org.jmdns:jmdns:3.5.7'
+    implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
```

Comment:
Let's not pull in all this Kotlin stuff for one tiny function. Please move `dexMetaKeyCapture()` to Game.java or another Java class.